### DISCUSSION TITLE
Search for packages with sorting by size

### DISCUSSION THREAD

**Discussion by HawkeyePierce89** [npm/feedback#746] [Website]
subj

Hi everyone!

What do you think about this? For example, I need calendar for my project, but I don't super-duper calendar with thousands options – only little simple calendar component.

<img width="1041" alt="изображение" src="https://user-images.githubusercontent.com/25603805/182900274-10999864-5472-4f54-8b9d-c816f2744f8b.png">

**Comment by ljharb** [comment]
What does "size" mean? Size on disk (compressed, or uncompressed)? Size in the bundle, which can't actually be accurately calculated without having your entire application's dependency graph, code, and bundler config?

**Reply by HawkeyePierce89** [reply]
at least this 

<img width="396" alt="изображение" src="https://user-images.githubusercontent.com/25603805/182902829-5b7f1939-084f-4cf7-b4b8-1fb17a7df39e.png">

**Reply by ljharb** [reply]
That's just size on disk tho - which is largely irrelevant, since in a node app disk space doesn't matter and in a bundle only the files you *use* matter.
